client/src/components/ProjectWizard/WizardPages/DiscoverTooltips.js

```diff
@@ -4,9 +4,9 @@ import { createUseStyles } from "react-jss";
 const useStyles = createUseStyles({
   container: {
     width: "calc((100% / 14.5) * 3);",
-    background: "#e9e9f1",
+    background: "#EEF1F4",
```

Comment:
Rather than hard-coding hex color values, this should get the colors.lightGray color value from the theme.